Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'webpacker', '~> 3.5'
+gem 'tailwindcss', '~> 0.2.0'
 
 group :development, :test do
   gem 'cucumber-rails', require: false
```

Comment:
fix indentation on 17 and 18

Reply:
displays correctly on ours locally, cannot change atm